salt/beacons/cert_info.py

```diff
@@ -100,11 +100,11 @@ def beacon(config):
 
         if isinstance(cert_path, dict):
             try:
-                notify_days = cert_path[cert_path.keys()[0]].get(
+                notify_days = cert_path[next(iter(cert_path.keys()))].get(
```

Comment:
Please remove the unnecessary call to `keys()` in all occurrences similar to this.

Reply:
I think these should all be done, like you said we just updated at the same time.